to_do_list/src/Components/AddButton.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import uuid from 'react-uuid';
+import AddIcon from "@material-ui/icons/Add";
+import SimpleDialog from './SimpleDialog';
+import '../Style/AddButton.css';
+
+function AddButton(props) {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = (shouldAdd, value) => {
+    if (shouldAdd){
+      props.add(uuid(), false, value);
+    }
+    setOpen(false);
+  };
+
+  return (
+    <div className='addButtonBox'>
+      <AddIcon className='addButton' onClick={handleClickOpen} />
+      <SimpleDialog open={open} onClose={handleClose} title="Add Assignment" action="Add"/>
```

Comment:
again, like in `assignment.js`, no need to render this component if you dont actually see it.